src/javascripts/ng-admin/Main/component/directive/maMenuBar.js

```diff
@@ -10,7 +10,10 @@ function maMenuBar($location, $rootScope, $compile) {
         link: function(scope, element) {
             scope.menu = scope.menu();
             scope.path = $location.path();
-            var openMenus = [];
+            // initialize openMenus
+            var openMenus = scope.menu.children().filter(function (menu) {
```

Comment:
It is not properly initialized. If we refresh on a link that open a child menu, the child menu will not be in the openMenus. And so the menu will close when navigating outside of the submenu even when autoClose is false.